packages/snaps-cli/src/webpack/utils.ts

```diff
@@ -120,6 +120,43 @@ export async function getDefaultLoader({
            * @see https://swc.rs/docs/configuration/compilation#jscparser
            */
           syntax: 'typescript',
+
+          /**
+           * This tells the parser to transpile JSX.
+           *
+           * @see https://swc.rs/docs/configuration/compilation#jscparser
+           * @see https://swc.rs/docs/configuration/compilation#jscparserjsx
+           */
+          tsx: true,
```

Comment:
Does this also work for jsx files?

Reply:
Just tested building a `.jsx` Snap and it works as expected.